src/command_modules/azure-cli-interactive/azclishell/telemetry.py

```diff
@@ -50,21 +79,11 @@ def start(self):
     def conclude(self):
         """ concludings recording stuff """
         self.end_time = str(datetime.datetime.now())
-        self.track_event('Run', {'start time': self.start_time,
-                                 'end time': self.end_time})
-
-        thread1 = TelThread(self.flush)
-        thread1.start()
-
-
-class TelThread(threading.Thread):
-    """ telemetry thread for exiting """
-    def __init__(self, threadfunc):
-        threading.Thread.__init__(self)
-        self.threadfunc = threadfunc
-
-    def run(self):
-        self.threadfunc()
+        run = json.dumps({'StartTime': str(self.start_time), 'EndTime': str(self.end_time)})
```

Comment:
Properties names' should follow specific formats: 
1. https://github.com/Azure/azure-cli/blob/master/src/azure-cli-core/azure/cli/core/telemetry.py#L185
2.https://github.com/Azure/azure-cli/blob/master/src/azure-cli-core/azure/cli/core/telemetry.py#L22